scripts/requirements.esp32.txt

```diff
@@ -10,3 +10,6 @@ pygdbmi<=0.9.0.2
 reedsolo>=1.5.3,<=1.5.4
 bitstring>=3.1.6
 ecdsa>=0.16.0
+kconfiglib==13.7.1
```

Comment:
This is the same as the file [`requirements.txt`](https://github.com/espressif/esp-idf/blob/v4.3/requirements.txt) in our ESP-IDF Release v4.3.

Reply:
Ok, can you fix it upstream?

Reply:
Hi @mspang, we will double check with our tools team on this specific requirement. Meantime, could we merge it as is, since it's located in `requirements.esp32.txt`, and we will update after our upsteam change is ready.

Reply:
I'll hold you to that.